File: new_server/vueblog-common/src/main/java/cn/smilex/vueblog/common/entity/common/VueBlogConfig.java
package cn.smilex.vueblog.common.entity.common;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author smilex
 * @date 2022/11/13/12:21
 * @since 1.0
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class VueBlogConfig {
    private String dataSourceJdbcUrl;
    private String dataSourceUserName;
    private String datasourcePassWord;
    private String jwtKey;
    private Integer jwtDayNumber;
    private Long vueBlogBeforeWebPageSize;
    private String vueBlogFriendDefaultHref;
    private String vueBlogFriendDefaultAvatar;
    private Long vueBlogAfterFriendPageSize;
    private String searchServer;
    private String searchKey;
    private String[] searchAttributesToHighlight;
    private Integer searchLimit;
    private String musicServer;
    private Boolean isProc;
}

File: new_server/vueblog-common/src/main/java/vip/smilex/vueblog/common/entity/common/VueBlogConfig.kt
package vip.smilex.vueblog.common.entity.common

import lombok.Data

/**
 * @author smilex
 * @date 2023/12/8 13:49:57
 */
@Data
data class VueBlogConfig(
    var dataSourceJdbcUrl: String,
    var dataSourceUserName: String,
    var datasourcePassWord: String,
    var jwtKey: String,
    var jwtDayNumber: Int,
    var vueBlogBeforeWebPageSize: Long,
    var vueBlogFriendDefaultHref: String,
    var vueBlogFriendDefaultAvatar: String,
    var vueBlogAfterFriendPageSize: Long,
    var searchServer: String,
    var searchKey: String,
    var searchAttributesToHighlight: Array<String>,
    var searchLimit: Int,
    var musicServer: String,
    var isProc: Boolean = false
)